src/frontend/src/helpers/Services.ts

```diff
@@ -106,12 +106,20 @@ namespace Services {
 
     // login successful
     await localStorage.connect(`seashell-${connection.username}`);
+    await localStorage.deleteDB();
+    await localStorage.connect(`seashell-${connection.username}`);
```

Comment:
Yeah but we'll forget about it :P.  Just use a different name -- it'll save everyone some trouble.